skyportal/plot.py

```diff
@@ -2386,6 +2410,29 @@ def make_spectrum_layout(obj, spectra, user, device, width, smoothing, smooth_nu
             shifting_elements.append(new_line)
             new_line.glyph.line_alpha = 1.0
 
+    new_line_w.visible = False
+    model_dict['custom_line'] = new_line_w
+    renderers.append(new_line_w)
+
+    w_input.js_on_change(
+        'value',
+        CustomJS(
+            args={'input': w_input, 'model_dict': model_dict},
+            code="""
+                    model_dict['custom_line'].data_source.data['x'][0] = input.value;
+                    model_dict['custom_line'].visible = true;
+                    model_dict['custom_line'].data_source.data['wavelength'][0] = input.value;
+                    model_dict['custom_line'].data_source.change.emit();
+                """,
```

Comment:
Is there a way to make the line not visible?

Reply:
you mean on the frontend ?

Reply:
Yeah. After I put in a value and see the line, can I make it disappear again or do I have to refresh the page?

Reply:
Currently, there is no button to make this line invisible but I can add it

Reply:
I would advocate that if the user puts in an empty value it should make the line disappear.

Reply:
ok I will try this

Reply:
I followed your advice and it's working well